telegram/bot.py

```diff
@@ -1430,7 +1493,7 @@ def getWebhookInfo(self, **kwargs):
 
         data = {}
 
-        result = self._request.post(url, data, timeout=kwargs.get('timeout'))
+        result = self._request.post(url, data, timeout=timeout)
```

Comment:
`timeout` can be `None` here, where the previous timeout where either ints or floats.
Is the behaviour the same for 0 and None?